### ISSUE TITLE
Lock UI

### ISSUE THREAD

**Issue by saraneth0o** [timebomb0/HordesUIMod#15]
Can you please add a button to lock the UI? I find that I end up clicking on and moving the unit frames far too often while fighting. 

Sorry if this isn't the place to ask for new features but I couldn't find out where.

Thanks for the work you are putting into this.

**Comment by timebomb0** [comment]
Good idea. I've just released version 1.3.3 of the mod, which fixes a couple things and makes it so you have to hold down UI frames for 1 second to move them, instead of 0.5 seconds. This should help lessen accidentally moving UI frames.

A lock UI button is still on my plate.
